juju/juju
# Juju doc update

## Please provide the following details to expedite Pull Request review:

---

## Description of change
Why is this change needed?
juju documentation update
For juju source code build/install reference
## QA steps
Not applicable
How do we verify that the change works?
Follow the steps modifiedin the README.md 
## Documentation changes

Does it affect current user workflow? CLI? API?
No

## Bug reference

Does this change fix a bug? Please add a link to it.
https://bugs.launchpad.net/bugs/1763201
